lib/note-content-editor.tsx

```diff
@@ -384,7 +385,9 @@ class NoteContentEditor extends Component<Props> {
       return [];
     }
 
-    const content = model.getValue().normalize().toLowerCase();
+    const content = removeAccents(
+      model?.getValue().normalize().toLowerCase() || ''
+    );
```

Comment:
see comment below: I believe that notes with lots of removed characters will end up shifting this too far and end up wrong/confusing. there's an open issue I think could already be related to this with checkboxes. (can't find it off hand).